packages/flutter_tools/lib/src/base/multi_root_file_system.dart

```diff
@@ -0,0 +1,273 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io
+    show
+        Directory,
+        File,
+        FileStat,
+        FileSystemEntity,
+        FileSystemEntityType,
+        Link;
+
+import 'package:file/file.dart';
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p; // flutter_ignore: package_path_import
```

Comment:
nit: import this `as path`;

Reply:
I can't, unfortunately. This would be an error otherwise:

```dart
  @override
  path.Context get path => _cachedPath ??= delegate.path;
```